tests/test_autonomy/test_cli/test_deploy/test_from_token.py

```diff
@@ -137,12 +135,10 @@ def test_from_token(
                 )
             )
 
-            out, err = capsys.readouterr()
-
-            assert result.exit_code == 0, out
-            assert "Service name: valory/oracle_hardhat" in out, err
-            assert "Building required images" in out, err
-            assert "Service build successful" in out, err
+            assert result.exit_code == 0, result.stdout
+            assert "Service name: valory/oracle_hardhat" in result.stdout
+            assert "Building required images" in result.stdout
+            assert "Service build successful" in result.stdout
```

Comment:
initialized with default parameters
https://github.com/valory-xyz/open-autonomy/blob/dc7748f4c2dd7a14f9b278c6aaeb59f734f3cba4/tests/test_autonomy/test_cli/base.py#L55
which means means that stderr is mixed with stdout ([reference](https://github.com/valory-xyz/open-autonomy/blob/dc7748f4c2dd7a14f9b278c6aaeb59f734f3cba4/tests/test_autonomy/test_cli/base.py#L55))